Engine/src/Application.bf

```diff
@@ -22,30 +29,100 @@ namespace SteelEngine
 			Dispose();
 		}
 
+		public void Dispose()
+		{
+			OnCleanup();
+		}
+
+		/// <summary>
+		/// Creates a new <see cref="SteelEngine.ECS.BaseSystem"/>. This operation is expensive, as it runs through all entities and registers viable ones to the new system.
+		/// Systems should be added as close to the start of the <see cref="SteelEngine.Application"/> as possible to avoid slowdowns.
+		/// </summary>
+		public BaseSystem CreateSystem<T>() where T : BaseSystem
+		{
+			let system = new T(this);
+			_systems.Add(system);
+
+			for (let item in Entity.EntityStore)
```

Comment:
Does anyone know if it is better to do `Entity.EntityStore.Values` if the `value` is the only thing really used? I prefer to have the whole `item` in case the `key` proves to be useful later.

Reply:
The value enumerator is little smaller and possibly does less work, since it doesn't have to fetch the key, but I don't think it makes any measurable difference in performance.